swap_meet/vendor.py

```diff
@@ -1,2 +1,83 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
+    def __init__(self, inventory = None):
+        if not inventory:
+            inventory = []
+            
+        self.inventory = inventory 
+
+    def add(self, add_item):
+        self.inventory.append(add_item)
+        return add_item
+
+    def remove(self, remove_item):
+        if remove_item in self.inventory:
+            self.inventory.remove(remove_item)
+            return remove_item
+        else:
+            return False 
```

Comment:
Consider first testing for the edge case, and then moving the main functionality of your function outside the `if/else`. This programming pattern is called a guard clause and can among other things enhance the readability of your code.

```suggestion
        if remove_item not in self.inventory:
            return False
        self.inventory.remove(remove_item)
        return remove_item
```